adev/src/content/examples/testing/src/app/demo/demo.spec.ts

```diff
@@ -1,5 +1,6 @@
 // #docplaster
 import {LightswitchComponent, MasterService, ValueService, ReversePipe} from './demo';
+import {firstValueFrom} from 'rxjs';
```

Comment:
```suggestion
import {firstValueFrom} from 'rxjs';
import {LightswitchComponent, MasterService, ValueService, ReversePipe} from './demo';
```

Import third party libs should be on first places, then internal imports.